### PR TITLE
ML baseline 3: Video Swin Transformer Kinetics 400 Swin-B ImageNet-22K

### PR DESCRIPTION
**Pull request by teresa-gao**
https://github.com/SwinTransformer/Video-Swin-Transformer

### CODE REVIEW

**Comment by teresa-gao** [comment]
I spent a lot of time this week trying to debug Video Swin Transformer to work with `train_job.py`.

- I implemented a workaround for the loss function and several other pieces in the body of `train_job.py`
- I ran into some errors in the forward/backward pass logic (`self.proj(x)` in `experiments/model_development/utils/models/Video_Swin_Transformer/mmaction/models/backbones/swin_transformer.py`) which looked deceptively similar to errors I'd encountered and successfully resolved with timesformer
- I ultimately abandoned efforts to resolve these errors
  - I first tried simple workarounds to massage the data into dimensions that the model seemed to expect, though this only introduced further errors down the line that could not be similarly resolved
  - I then re-consulted [the original paper](https://arxiv.org/abs/2106.13230) in hopes that it would help clear up my understanding, though my confusions seemed more based in the nuances of `pytorch` and the code itself than the underlying math

As next steps, I'm planning to revert my hack-ier changes to the repo and reimplement `train_job.py` with an emphasis on preserving the original Video Transformer Code over the structure of the original `train_job.py` file.

- Using the downloaded pretrained model for Video Swin Transformer Swin-B ImageNet-22K Kinetics 400, I'm hoping that I can use the corresponding [config] as an argument to `train_model()` in `mmaction/apis/train.py`
- My prior partially functional code used `init_recognizer()`, also from `mmaction/apis`; I'm hoping that this in conjunction with `train_model()` and/or similar code as to `main()` in `tools/train.py` should be similar enough to the former `train_job.py`

**Comment by teresa-gao** [comment]
replaced by #23
